cluster-soak/src/main/resources/application.conf

```diff
@@ -25,3 +25,5 @@ akka {
   }
 }
 
+# Required for the response from the K8s API server
```

Comment:
Hmm should we document this somewhere? Do we have a 'troubleshooting' section in the akka-management docs yet?

Reply:
No i'll do that now

Reply:
https://github.com/akka/akka-management/pull/524 :+1: